mmv1/products/edgenetwork/Subnet.yaml

```diff
@@ -0,0 +1,137 @@
+    vars:
+      edgenetwork_subnet_id: 'example-subnet-with-vlan-id'
+      edgenetwork_network_id: 'example-network'
+    test_env_vars:
+      location: :REGION
+      zone: :ZONE
+parameters:
+  - !ruby/object:Api::Type::String
+    name: 'location'
+    required: true
+    immutable: true
+    url_param_only: true
+    description: |
+      The Google Cloud region to which the target Distributed Cloud Edge zone belongs.
+  - !ruby/object:Api::Type::String
+    name: 'zone'
+    required: true
+    immutable: true
+    url_param_only: true
+    description: |
+      The name of the target Distributed Cloud Edge zone.
+  - !ruby/object:Api::Type::String
+    name: 'subnet_id'
+    required: true
+    immutable: true
+    url_param_only: true
+    description: |
+      A unique ID that identifies this subnet.
+properties:
+  - !ruby/object:Api::Type::String
+    name: 'name'
+    output: true
+    description: |
+      The canonical name of this resource, with format
+      `projects/{{project}}/locations/{{location}}/zones/{{zone}}/subnets/{{subnet_id}}`
+  - !ruby/object:Api::Type::KeyValuePairs
+    name: 'labels'
+    required: false
+    description: |
+      Labels associated with this resource.
+  - !ruby/object:Api::Type::String
+    name: 'description'
+    required: false
+    description: |
+      A free-text description of the resource. Max length 1024 characters.
+  - !ruby/object:Api::Type::String
+    name: 'createTime'
+    output: true
+    description: |
+      The time when the subnet was created.
+      A timestamp in RFC3339 UTC "Zulu" format, with nanosecond resolution and up to nine
+      fractional digits. Examples: `2014-10-02T15:01:23Z` and `2014-10-02T15:01:23.045123456Z`.
+  - !ruby/object:Api::Type::String
+    name: 'updateTime'
+    output: true
+    description: |
+      The time when the subnet was last updated.
+      A timestamp in RFC3339 UTC "Zulu" format, with nanosecond resolution and up to nine
+      fractional digits. Examples: `2014-10-02T15:01:23Z` and `2014-10-02T15:01:23.045123456Z`.
+  - !ruby/object:Api::Type::ResourceRef
+    name: 'network'
+    resource: 'Network'
+    imports: 'name'
+    required: true
+    custom_expand: 'templates/terraform/custom_expand/resourceref_with_validation.go.erb'
+    description: |
+      The ID of the network to which this router belongs.
+      Must be of the form: `projects/{{project}}/locations/{{location}}/zones/{{zone}}/networks/{{network_id}}`
+  - !ruby/object:Api::Type::Array
+    name: 'ipv4Cidr'
+    item_type: Api::Type::String
+    description: |
+      The ranges of ipv4 addresses that are owned by this subnetwork, in CIDR format.
+  - !ruby/object:Api::Type::Array
+    name: 'ipv6Cidr'
+    item_type: Api::Type::String
+    description: |
+      The ranges of ipv6 addresses that are owned by this subnetwork, in CIDR format.
+  - !ruby/object:Api::Type::Integer
+    name: 'vlanId'
+    default_from_api: true
+    send_empty_value: false
```

Comment:
What is the reason to add send_empty_value: false here?

Reply:
Is it false by default? I will remove that line if so.

Reply:
Yeah, false is by default.

Reply:
Done